must-gather/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM quay.io/openshift/origin-must-gather:4.11.0
```

Comment:
Can we use the production base image?

Reply:
also 4.11 is very old

Reply:
Latest Dockerfile template will be in hack repo